packages/shared/src/components/auth/CodeVerificationForm.tsx

```diff
@@ -31,7 +31,7 @@ function CodeVerificationForm({
   const [emailSent, setEmailSent] = useState(false);
   const { sendEmail, verifyCode, resendTimer, isLoading } = useAccountEmailFlow(
     {
-      flow: AuthFlow.Recovery,
+      flow: AuthFlow.Verification,
```

Comment:
Just checking that we don't wan't these other events anymore but are replacing these with these new ones.

Reply:
Thanks for raising this, checked with Dave and we decided to add some more/different ones.